### PR TITLE
[update] README.md, add waylevel

### PR DESCRIPTION
**Pull request by Shinyzenith**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -246,9 +246,10 @@ No Wayland-specific requirements, so you can use your xorg solution of choice to
 ## Tools
 
 * [lswt](https://git.sr.ht/~leon_plickat/lswt) - List Wayland toplevels in both human readable and machine parsable formats
+* [waylevel](https://git.sr.ht/~shinyzenith/waylevel) - A simple debugging tool which lists compositor specific information.
+* [wev](https://git.sr.ht/~sircmpwn/wev) - A tool for debugging events on a Wayland window, analagous to the X11 tool xev
```
Comment (apiraino) [inline README.md:250]:
```suggestion
* [wev](https://git.sr.ht/~sircmpwn/wev) - A tool for debugging events on a Wayland window, analogous to the X11 tool xev
```

**Comment by apiraino** [comment]
great tools, thanks for the suggestion!

**Comment by Shinyzenith** [comment]
> great tools, thanks for the suggestion!

Glad to help!

**Comment by natpen** [comment]
Thank you!

**Pull request merged**